src/items.md

```diff
@@ -1,5 +1,42 @@
 # Items
 
+> **<sup>Syntax:<sup>**\
+> _Item_:\
+> &nbsp;&nbsp; [_OuterAttribute_]<sup>\*</sup> [_Visibility_]<sup>?</sup>\
+> &nbsp;&nbsp; (\
+> &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp;  [_Module_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_ExternCrate_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_UseDeclaration_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_Function_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_TypeAlias_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_Struct_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_Enumeration_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_Union_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_ConstantItem_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_StaticItem_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_Trait_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_Implementation_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | [_ExternBlock_]\
+> &nbsp;&nbsp; &nbsp;&nbsp; | _Macro_\
+> &nbsp;&nbsp; &nbsp;&nbsp; | _MacroDefinition_\
+> &nbsp;&nbsp; )
+
+[_OuterAttribute_]: attributes.html
```

Comment:
All links at the bottom of the page, sorted alphabetically (by ascii/unicode codepoint)